api/src/ic_api/store.py

```
@@ -256,25 +257,64 @@ def build_default_store() -> SessionStore:
         try:
             from ic_api.db_store import PersistentSessionStore
 
-            return PersistentSessionStore(dsn)
+            store = PersistentSessionStore(dsn)
+            _backend_name, _backend_persistent = "postgres", True
+            print(
+                "[ic_api.store] session store: postgres (sessions survive a "
+                "restart).",
+                file=sys.stderr,
+            )
+            return store
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)